components/list/list-item-drag-mixin.js

```diff
@@ -24,6 +81,11 @@ export const ListItemDragMixin = superclass => class extends superclass {
 		}
 	}
 
+	firstUpdated(changedProperties) {
+		this.addEventListener('dragenter', this._hostDragEnter);
```

Comment:
This makes the entire host a the ability to do things when someone first enters it. This is used to show the better drop targets that block other pointer events. So thats why they can only show up when something is being dragged